net/wireguard-tools/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wireguard-tools
+# please only update version when the tools have been modified/updated,
+# since the Linux module aspect isn't of utility for macOS
+version             0.0.20180613
+categories          net
+platforms           darwin
+license             GPL-2
+maintainers         {isi.edu:calvin @cardi} openmaintainer
+description         Tools for the WireGuard VPN
+long_description    \
+    WireGuard-tools contains command-line tools to interact with \
+    the userspace Go implementation of WireGuard. Currently there \
+    are two tools: wg, to set and retrieve configuration of \
+    WireGuard interfaces, and wg-quick, set up a WireGuard interface \
+    simply.
+
+homepage            https://www.wireguard.com/
+master_sites        https://git.zx2c4.com/WireGuard/snapshot/
+distname            WireGuard-${version}
+use_xz              yes
+
+checksums           rmd160  2df3459bc35e0598fea7f2e662132e331a414ca2 \
+                    sha256  c120cdedc3967dcb4ad5c1c7eadd2a1b04ef5dbf2fe60cc8e7c0db337bcda7dc \
+                    size    269316
+
+depends_run         port:bash \
+                    port:wireguard-go
+
+# remove for version > 0.0.20180613
+# upstream commit from 14 Jun 2018 "tools: getentropy requires 10.12"
+# https://git.zx2c4.com/WireGuard/patch/?id=5bb62fe22f45b5b5deef4db23ae47c95e1679d1d
+# (MacPorts doesn't handle that URL properly, so replicate it locally)
+patchfiles          patch-genkey.c.diff
+patch.pre_args      -p1
+
+use_configure       no
+
+# only build and install the tools for macOS
+build.pre_args      -C src/tools
+build.target
+
+destroot.pre_args   -C src/tools
+destroot.args       install
+destroot.post_args-append PREFIX=${prefix} \
+                    SYSCONFDIR=${prefix}/etc \
+                    WITH_BASHCOMPLETION=no \
```

Comment:
Note that MacPorts does support bash completion files, and I'm not sure it is worth having a variant for it given that it doesn't harm anyone who doesn't use it.

Reply:
Line 55 in the bash_completion variant will install the `bash-completion` port, which some users may not want by default.

Is it OK then to install the bash completion files without having the port? I will remove the variant, install the bash completion files, and not require `bash-completion` as a dependency.

Reply:
> Is it OK then to install the bash completion files without [depending on] the [bash-completion] port?

Yes, I think so. (Others may disagree with me, but many ports seem to do this, so I think it is fine.)

Reply:
Okay, great–I'm okay with either, just wasn't sure what the standard was.

I've pushed fixes for the bash completion and your other comments.